SYMLINK_CACHE.json

```diff
@@ -0,0 +1 @@
+{"script": {"ansible": ["ansible-config", "ansible-console", "ansible-doc", "ansible-galaxy", "ansible-inventory", "ansible-playbook", "ansible-pull", "ansible-vault"]}, "library": {"ec2_metadata_facts.py": ["ansible/modules/cloud/amazon/_ec2_facts.py"], "ec2_vpc_dhcp_option.py": ["ansible/modules/cloud/amazon/_ec2_vpc_dhcp_options.py"], "ec2_vpc_dhcp_option_facts.py": ["ansible/modules/cloud/amazon/_ec2_vpc_dhcp_options_facts.py"], "iam_server_certificate_facts.py": ["ansible/modules/cloud/amazon/_iam_cert_facts.py"], "aws_s3.py": ["ansible/modules/cloud/amazon/_s3.py"], "os_server_action.py": ["ansible/modules/cloud/openstack/_os_server_actions.py"], "ovirt_affinity_group.py": ["ansible/modules/cloud/ovirt/_ovirt_affinity_groups.py"], "ovirt_affinity_label.py": ["ansible/modules/cloud/ovirt/_ovirt_affinity_labels.py"], "ovirt_affinity_label_facts.py": ["ansible/modules/cloud/ovirt/_ovirt_affinity_labels_facts.py"], "ovirt_cluster.py": ["ansible/modules/cloud/ovirt/_ovirt_clusters.py"], "ovirt_cluster_facts.py": ["ansible/modules/cloud/ovirt/_ovirt_clusters_facts.py"], "ovirt_datacenter.py": ["ansible/modules/cloud/ovirt/_ovirt_datacenters.py"], "ovirt_datacenter_facts.py": ["ansible/modules/cloud/ovirt/_ovirt_datacenters_facts.py"], "ovirt_disk.py": ["ansible/modules/cloud/ovirt/_ovirt_disks.py"], "ovirt_external_provider.py": ["ansible/modules/cloud/ovirt/_ovirt_external_providers.py"], "ovirt_external_provider_facts.py": ["ansible/modules/cloud/ovirt/_ovirt_external_providers_facts.py"], "ovirt_group.py": ["ansible/modules/cloud/ovirt/_ovirt_groups.py"], "ovirt_group_facts.py": ["ansible/modules/cloud/ovirt/_ovirt_groups_facts.py"], "k8s.py": ["ansible/modules/clustering/k8s/_k8s_raw.py", "ansible/plugins/lookup/_openshift.py"], "aci_interface_policy_fc.py": ["ansible/modules/network/aci/_aci_intf_policy_fc.py"], "aci_interface_policy_l2.py": ["ansible/modules/network/aci/_aci_intf_policy_l2.py"], "aci_interface_policy_lldp.py": ["ansible/modules/network/aci/_aci_intf_policy_lldp.py"], "aci_interface_policy_mcp.py": ["ansible/modules/network/aci/_aci_intf_policy_mcp.py"], "aci_interface_policy_port_channel.py": ["ansible/modules/network/aci/_aci_intf_policy_port_channel.py"], "aci_interface_policy_port_security.py": ["ansible/modules/network/aci/_aci_intf_policy_port_security.py"], "say.py": ["ansible/modules/notification/_osx_say.py"], "acme_certificate.py": ["ansible/modules/web_infrastructure/_letsencrypt.py"]}}
```

Comment:
This appears to have been added accidentally.

Reply:
Oops. Why isn't it gitignorred?

Reply:
It should be.